api.go

```diff
@@ -77,6 +77,13 @@ type LineMatch struct {
 	LineFragments []LineFragmentMatch
 }
 
+type Symbol struct {
+	Sym        string
+	Kind       string
+	Parent     string
```

Comment:
ctags calls this a scope. Why do you call it a Parent? I think we call it a parent in the sourcegraph API, but I am not sure if that is the best name here. WDYT?

Reply:
Yeah, I was a little confused with the inconsistency between Zoekt and SG naming. Upstream Zoekt also uses `ParentType` instead of `ScopeKind` which I found strange. I changed it to `ParentKind` in this PR.

s/Parent/Scope and s/Type/Kind are probably the best naming choices here since it is what ctags follows.